src/main/java/racingcar/stringcalculator/StringCalculator.java

```diff
@@ -0,0 +1,64 @@
+package racingcar.stringcalculator;
+
+/**
+ * Created : 2020-10-29 오후 2:33
+ * Developer : Seo
+ */
+public class StringCalculator {
+
+    public int plus(String s1, String s2) {
+        return Integer.parseInt(s1) + Integer.parseInt(s2);
+    }
+
+    public int minus(String s1, String s2) {
+        return Integer.parseInt(s1) - Integer.parseInt(s2);
+    }
+
+    public int multiply(String s1, String s2) {
+        return Integer.parseInt(s1) * Integer.parseInt(s2);
+    }
+
+    public int divide(String s1, String s2) {
+        return Integer.parseInt(s1) / Integer.parseInt(s2);
+    }
+
+    public boolean isOperator(String s) {
+        if (!"+".equals(s) && !"-".equals(s) && !"*".equals(s) && !"/".equals(s)) {
+            throw new IllegalArgumentException();
```

Comment:
```suggestion
            throw new IllegalArgumentException("상세 메세지");
```
Exception에 대한 상세 메세지를 작성해봐도 좋을거 같아요! 😄